src/scores/continuous/quantile_loss_impl.py

```diff
@@ -17,7 +17,7 @@ def quantile_score(
     *,  # Force keywords arguments to be keyword-only
     reduce_dims: Optional[Sequence[str]] = None,
     preserve_dims: Optional[Sequence[str]] = None,
```

Comment:
Should this be using `FlexibleDimensionType`?

Reply:
Yes, I'd say so. I'm folding that into the tidyups MR now.

Reply:
Yes, thanks, good pickup

Reply:
FYI, fixes pushed into the final MR so I can confirm nothing new is revealed and all things are finally resolved.